include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -82,6 +82,12 @@
 #    define _ONEDPL_PRAGMA(x) _Pragma(#    x)
 #endif
 
+// For Windows, define NOMINMAX to avoid definition of min and max macros inside of windows.h which clash with STL
+#ifdef _MSC_VER && !defined(NOMINMAX)
+#    define NOMINMAX
+#    define _ONEDPL_DEFINED_NOMINMAX 1
```

Comment:
Is this still needed? I don't see it used elsewhere.

Reply:
It is not needed directly.  I added it as a way to check if oneDPL was the one who added it.  

I could see it being useful somehow in debugging a case, but perhaps not.  I wouldn't object to removing it.

Reply:
No problem from my side.